AppCenter/AppCenter/Internals/Context/Device/MSDeviceTracker.m

```diff
@@ -57,8 +57,9 @@ - (instancetype)init {
 
     // Restore past sessions from NSUserDefaults.
     NSData *devices = [MS_APP_CENTER_USER_DEFAULTS objectForKey:kMSPastDevicesKey];
```

Comment:
I can see running the puppet app on macOS still gives me `"sdkName" : "appcenter.ios"` in reported device info.

Reply:
Since Catalyst version is based on iOS sdk I think it is expected, isn't it?
What `sdkName` do you expect in this case?

Reply:
Discussed this offline and `appcenter.ios` should be correct since it's an ios sdk. But it was reported differently for SPM so I created a PR to fix that: https://github.com/microsoft/appcenter-sdk-apple/pull/2089

Reply:
It should be `appcenter.maccatalyst`, same code base than iOS but different code path. I can see your PR is taking car of that 👍 .